### PR TITLE
fix RVGate doc

### LINKED ISSUE #11856: Documentation of RVGate is incorrect
### Environment

N/A

### What is happening?

Received this in an email:
>Hi, I think I found some errors in the Qiskit documentation at
<https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.RVGate>
and I'm contacting you because you look like the two people who most recently edited the source file at
<https://github.com/Qiskit/qiskit/blob/stable/0.46/qiskit/circuit/library/generalized_gates/rv.py>
The matrix representation given in the documentation seems to be wrong.  I compared it to the definition given in
<https://arxiv.org/pdf/2104.14875.pdf>
on page 4, equation 1, we see the definition of the rotation matrix.  It almost matches the definition given in the documentation at
<https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.RVGate>
except for two mistakes: the "sinc" function should be "sin", and the angle should be divided by two.  This can be compared to the source code at
<https://github.com/Qiskit/qiskit/blob/stable/0.46/qiskit/circuit/library/generalized_gates/rv.py>
at lines 86 and 87, where we see the angle divided by two, and we see the use of the sin and cos functions.

### How can we reproduce the issue?

N/A

### What should happen?

N/A

### Any suggestions?

_No response_

### PR DESCRIPTION
**Pull request by kevinsung**
### Summary
Fixes #11856

### CODE REVIEW

**Comment by kevinsung** [comment]
CI failure seems unrelated.

File: qiskit/circuit/library/generalized_gates/rv.py [hunk 1]
```diff
@@ -36,14 +36,17 @@ class RVGate(Gate):
 
     .. math::
 
-        \newcommand{\rotationangle}{|\vec{v}|}
-        \newcommand{\sinc}{\text{sinc}}
+        \newcommand{\rotationangle}{\frac{|\vec{v}|}{2}}
             R(\vec{v}) = e^{-i \vec{v}\cdot\vec{\sigma}} =
```
Comment (Cryoris) [inline qiskit/circuit/library/generalized_gates/rv.py]: Should the factor 1/2 also be in the exponent here? In the paper that was referenced they also have it I think
```suggestion
            R(\vec{v}) = e^{-i/2 \vec{v}\cdot\vec{\sigma}} =
```

File: qiskit/circuit/library/generalized_gates/rv.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 
 class RVGate(Gate):
-    r"""Rotation around arbitrary rotation axis :math:`v` where :math:`|v|` is
+    r"""Rotation around arbitrary rotation axis :math:`\vec{v}` where :math:`|\vec{v}|` is
```
Comment (Cryoris) [inline qiskit/circuit/library/generalized_gates/rv.py]: While we're at it, could you add the used norm for the vector? Same for the definition below
```suggestion
    r"""Rotation around arbitrary rotation axis :math:`\vec{v}` where :math:`\|\vec{v}\|_2` is
```

**Review by Cryoris — COMMENTED** [review]
Oh that's a good catch! I left two comments below.

File: qiskit/circuit/library/generalized_gates/rv.py (hunk 1 above)
Comment (kevinsung) [reply qiskit/circuit/library/generalized_gates/rv.py]: Done.

File: qiskit/circuit/library/generalized_gates/rv.py (hunk 2 above)
Comment (kevinsung) [reply qiskit/circuit/library/generalized_gates/rv.py]: Done.

**Review by 1ucian0 — APPROVED** [review]

**Comment by 1ucian0** [comment]
@mergifyio backport stable/0.46

**Pull request merged**